2-mandatory/1-weather-report.js

```diff
@@ -13,9 +13,25 @@
 
 function getTemperatureReport(cities) {
     // TODO
-}
-
+    let arr = [];
```

Comment:
Your implementation here looks good 👍 
You are adding the value returned by `temperatureService` to the array `arr`.
The question wants you to take this value, and put it into a string - like "The temperature in London is 10 degrees".
Can you think of how to create a string like this using the value you get back from the `temperatureService`?
One idea is to use template literals in JavaScript: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals